src/ORM/Rule/IsUnique.php

```diff
@@ -53,6 +53,11 @@ public function __invoke(EntityInterface $entity, array $options)
             return true;
         }
 
+        $allowMultipleNulls = true;
+        if (isset($options['allowMultipleNulls'])) {
```

Comment:
@dereuromark Theoretically, I think you are right. But as you know, we are using string literals in many places. For example:
```php
$validator
    ->add('title', 'notEmpty', [
        'rule' => 'notEmpty',
        'message' => __('You need to provide a title'),
    ]);
```